system/tests/kohana/MinionTaskTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * Tests the Minion library
+ *
+ * @group kohana
+ * @group kohana.core
+ * @group kohana.core.config
+ *
+ * @package    Koseven
+ * @category   Tests
+ * @author     Koseven Team
+ * @author     Piotr Gołasz <pgolasz@gmail.com>
+ * @copyright  (c) Kohana Team
+ * @license    https://koseven.ga/LICENSE.md
+ */
+class MinionTaskTest extends Unittest_TestCase
+{
+
+	/**
+	 * Tests that Minion Task Help works assuming all other tasks work aswell
+	 */
+	public function test_minion_runnable()
+	{
+		$minionResponse = Minion_Task::factory(['task' => 'help']);
```

Comment:
$minionResponse does not conform with required coding style. It should be $minion_response.